pkg/api/constant.go

```diff
@@ -18,8 +18,10 @@ const (
 	// DPTPRequesterLabel is the label on a Kubernates CR whose value indicates the automated tool that requests the CR
 	DPTPRequesterLabel = "dptp.openshift.io/requester"
 
-	KVMDeviceLabel = "devices.kubevirt.io/kvm"
-	ClusterLabel   = "ci-operator.openshift.io/cluster"
+	KVMDeviceLabel      = "devices.kubevirt.io/kvm"
+	ClusterLabel        = "ci-operator.openshift.io/cluster"
+	CLusterProfileLabel = "ci-operator.openshift.io/cloud-profile"
```

Comment:
Let's call it `cloud-cluster-profile` so that its relation to `cluster_profile` is a little bit more obvious.

Reply:
Updated.